Controller/Annotations/AbstractScalarParam.php

```diff
@@ -63,6 +64,12 @@ public function getConstraints()
             $constraints = array(
                 new All(array('constraints' => $constraints)),
             );
+            if (false === $this->nullable) {
```

Comment:
Isn't this something that should be added when `$this->map` is `false` too?

Reply:
If `$this->map` is `false` then it's added [here](https://github.com/FriendsOfSymfony/FOSRestBundle/blob/da8d1ff7d91e5e178022b82a206eacf9b604f3b1/Controller/Annotations/AbstractParam.php#L69).